aten/src/ATen/cuda/CUDASparseDescriptors.h

```diff
@@ -42,6 +42,12 @@ class TORCH_CUDA_CPP_API CuSparseDnMatDescriptor
   CuSparseDnMatDescriptor(const Tensor& input);
 };
 
+class TORCH_CUDA_CPP_API CuSparseDnVecDescriptor
+    : public CuSparseDescriptor<cusparseDnVecDescr, &cusparseDestroyDnVec> {
+ public:
+  CuSparseDnVecDescriptor(const Tensor& input);
```

Comment:
In general you might want to make this constructors explicit. To quote our linter "A single-argument constructor may inadvertently be used as a type conversion constructor."